@xen-orchestra/lite/src/components/infra/InfraHostItem.vue

```diff
@@ -42,6 +51,9 @@ const isCurrentHost = computed(
   () => props.hostOpaqueRef === uiStore.currentHostOpaqueRef
 );
 const [isExpanded, toggle] = useToggle();
+
+const isTooltipDisabled = (target: HTMLElement) =>
+  !hasEllipsis(target.querySelector(".text"));
```

Comment:
How about moving this function in `/utils` or `tooltip.store` to avoid duplication?

Reply:
I'm not really sure. Could we wait to see if it happens this often?